doc/source/extending-jupyterhub.rst

```diff
@@ -176,3 +176,26 @@ report the current ``Ready/NotReady`` status of all nodes in the cluster.
      size for the workshop. This workflow also helps you avoid scrambling on
      the workshop day to set up the cluster and JupyterHub.
    - **After the workshop:** The cluster can be deleted.
+
+
+Setting up HTTPS
+----------------
+
+Zero to JupyterHub makes setting up HTTPS quite easy.
```

Comment:
Maybe a sentence with a quick reason you might want to do this. 

Also shouldn't we call this the JupyterHub helm chart, instead of Zero to JupyterHub? Or do we think of the helm chart *as* z2jh?

E.g. "HTTPS allows you to XXX. The JupyterHub helm chart makes setting up HTTPS quite easy."

Reply:
HTTPS allows you to secure your authentication flows.

That's the specific need we have for Globus Auth, since it requires an HTTPS callback URL.

Reply:
I think of the chart as z2jh, mostly by dint of the fact that that's what I've been calling it in talks and what not...